COLLABORATOR_GUIDE.md

```diff
@@ -237,7 +237,8 @@ Save the file and close the editor. You'll be asked to enter a new
 commit message for that commit. This is a good moment to fix incorrect
 commit logs, ensure that they are properly formatted, and add
 `Reviewed-By` lines.
-* The commit message text must conform to the [commit message guidelines](../CONTRIBUTING.md#step-3-commit).
+* The commit message text must conform to the
+[commit message guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit).
```

Comment:
Tiniest of nits but might it be better to use one of the relative URLS `./CONTRIBUTING.md#step-3-commit` or `CONTRIBUTING.md#step-3-commit` rather than the FQDN etc.? This way, if you're looking at the doc on a different branch, the link goes to the doc as it exists on that branch rather than to master. The problem with the current link is the `..` at the start.